pkg/block/s3/adapter.go

```diff
@@ -473,7 +473,7 @@ func (a *Adapter) GetRange(ctx context.Context, obj block.ObjectPointer, startPo
 		}).Error("failed to get S3 object range")
 		return nil, err
 	}
-	sizeBytes = objectOutput.ContentLength
+	sizeBytes = *objectOutput.ContentLength
```

Comment:
Can you use the aws.* helper function to extract the value as it will check for nil